src/Ship.h

```diff
@@ -334,6 +336,9 @@ class Ship : public DynamicBody {
 
 	double m_hydrogenScoopedAccumulator = 0;
 
+	double m_latestSpawnTime = 0.0;
+	std::deque<CargoBody*> m_cargoSpawnQueue;
```

Comment:
Stylistic nitpick: pointer/reference are right-aligned, so there should be a space after `CargoBody`.